### ISSUE TITLE
Gaps in data collection obfuscate graphs

### ISSUE THREAD

**Issue by matejak** [marcusburghardt/CommunityMon#2]
When the time series isn't contiguous, it splits into two (or possibly more) chunks that are treated separately, as different quantities. For example, on the image below, there is a duplicated "Open PRs" quantity, the red one is before the gap, orange one after the gap, and it obfuscates the summary.

![image](https://user-images.githubusercontent.com/2823847/172584381-b35d4475-c8be-4b31-89fd-f6f70d0a1b75.png)

What would help:

- Unify colors in the graph, so a quantity is represented by a distinct color
- Consider only the last chunk when creating the pie graph summary

**Comment by marcusburghardt** [comment]
Thanks for reporting this @matejak.

This gaps appears because there was no metric found in the Prometheus database during that specific time frame. If I am not wrong, it is possible to workaround this on Grafana side, repeating the last available information when a gap is found.

Regarding the duplication of metrics, it is a known issue related to data imported from another instance, where the Prometheus job id had a different name. I will check if this could be fixed on Prometheus database.

**Comment by marcusburghardt** [comment]
I found a much easier way to fix this problem on Grafana side, just improving the PromQL queries.

As clarification to complement my last comment, regarding the gaps, I had already defined the Grafana charts to connect points between gaps. However, this was not happening in this case due to the second problem I mentioned.

I fixed the PromQL to treat the metric as the same even if the relevant label was renamed.

Fixed by this commit:
https://github.com/marcusburghardt/CommunityMon/commit/5e6bdbf4d2683d1a3476732cb2e543bb1c8b7000
